IPs/RSKIP251.md

```diff
@@ -0,0 +1,71 @@
+# Support eth_getProof JSON RPC method
+
+|RSKIP          |251           |
+| :------------ |:-------------|
+|**Title**      |Support eth_getProof JSON RPC method |
+|**Created**    |30-06-2021 |
+|**Author**     |FJ |
+|**Purpose**    |Usa,Sec |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+RSK implements a Unitrie data structure to store all the relevant blockchain data, such as account states and storage values, this particular data structure enables an easy verification of each stored value, just by getting a merkle proof. Currently these proofs are only useful for blockchain nodes, but in order to allow verification of accounts outside the client, we need an additional function delivering us the required proof. 
+
+Combined with a stateRoot (from the blockheader) it enables offline verification of any account or storage-value. 
```

Comment:
1. This sentence should be part of the above paragraph.. the "it"  in "it enables" refers to the function mentioned above
2. ... enables "offline verification" .. what does "offline" mean?
3. Even though it is mentioned in title (and motivation), it is good to explicitly say this is a RPC method in the abstract